aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/MutationProcessor.java

```diff
@@ -97,10 +99,14 @@ void startDrainingMutationOutbox() {
             .flatMapCompletable(merger::merge)
             // Lastly, remove the item from the outbox, so we don't process it again.
             .andThen(mutationOutbox.remove(mutationOutboxItem))
-            .andThen(Completable.fromAction(() -> {
-                LOG.info("Pending mutation was published up to Cloud: " + mutationOutboxItem);
+            .doOnComplete(() -> {
+                LOG.verbose(
+                    "Pending mutation was published to cloud successfully, " +
+                    "and removed from the mutation outbox: " + mutationOutboxItem
```

Comment:
Do we usually indent string concatenation additional lines? Thought I saw that in other PRs.

Reply:
Probably should. I'll do it, here.